automatic/peazip/tools/chocolateyinstall.ps1

```diff
@@ -2,22 +2,16 @@
 
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
-$fileName = if ((Get-ProcessorBits 64) -and ($env:chocolateyForceX86 -ne 'true')) {
-  Write-Host "Installing 64 bit version" ; 'peazip-6.5.0.WIN64.exe' # 64-bit
-} else {
-  Write-Host "Installing 32 bit version" ; 'peazip-6.5.0.WINDOWS.exe' # 32-bit
-}
-
 $packageArgs = @{
-  packageName    = 'peazip'
+  file           = "$toolsDir\peazip-6.5.0.WINDOWS.exe"
+  file64         = "$toolsDir\peazip-6.5.0.WIN64.exe"
   fileType       = 'exe'
+  packageName    = 'peazip'
   softwareName   = 'PeaZip'
-  file           = "$toolsDir\$fileName"
   silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
   validExitCodes = @(0, 1223)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
-# Lets remove the installer as there is no more need for it
-Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
+ls $toolsDir\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" } }
```

Comment:
Aliases shouldn't be used in scripts.

Reply:
[1.1.7](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#117-clean-code) allows standard aliases.

Reply:
*Use aliases sparingly. **If you use them, limit their use to the default aliases which are read-only (can not be modified by the user)**, well known, and will be supported in the future versions of Powershell. Also ensure they are compatible with PowerShell v2+.* (my emphasis)

1,1.7 allows aliases that are read-only. `ls`, `rm` and `sc` are not read-only. `%` and `?` are.

Reply:
That is then error in guidelines.